src/client/pages/ToeristischeVerhuurDetail/VergunningVerhuur.tsx

```diff
@@ -51,13 +51,17 @@ function useStatusLineItems(
           vergunning.caseType === 'B&B - vergunning'
             ? 'In behandeling'
             : 'Verleend',
-        datePublished: vergunning.dateRequest,
+        datePublished:
+          vergunning.status === 'Afgehandeld' ||
+          vergunning.status === 'In behandeling'
+            ? vergunning.dateRequest
```

Comment:
dateRequest kan hier dus alleen staan wanneer het aan vakantieverhuurvergunning is. Voor B&B tonen we vooralsnog geen datum bij In behandeling.